### ISSUE TITLE
Pagination element -  Background and foreground colors do not have a sufficient contrast ratio.

### ISSUE THREAD

**Issue by spelkey-ucd** [ucd-library/ucdlib-theme#105]
Greyed out buttons need to be darker

**Comment by wrenaria** [comment]
Is this for inactive buttons? If so, my understanding is they do not need to meet contrast minimums.

> **Inactive User Interface Components**
> User Interface Components that are not available for user interaction (e.g., a disabled control in HTML) are not required to meet contrast requirements. An inactive user interface component is visible but not currently operable. An example would be a submit button at the bottom of a form that is visible but cannot be activated until all the required fields in the form are completed.

Source: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

**Comment by spelkey-ucd** [comment]
Yeah, it is the disabled links. hmm maybe it needs a `aria-disabled` property on it too.

**Comment by wrenaria** [comment]
The problem with making disabled items meet color contrast minimums is the minimum accepted colors are so dark that the disabled items stop reading as disabled to standard sighted users because there is not enough contrast been the disabled version and functional version. Dark gray, for example, is not an obvious enough difference from black.

**Comment by spelkey-ucd** [comment]
when on page 1, the "Prev" button is actually an `a` tag with `pointer-events: none`. This is where the problem lies. it should be a disabled button instead, which will allow it to be ignored by color contrast tests.

As a separate issue, the `...` `a` tag needs an aria-label
